v2/pkg/engine/datasource/graphql_datasource/graphql_datasource.go

```diff
@@ -1274,6 +1284,49 @@ func (p *Planner[T]) printQueryPlan(operation *ast.Document) {
 	p.debugPrintln(args...)
 }
 
+func (p *Planner[T]) generateQueryPlansForFetchConfiguration(operation *ast.Document) {
+	if !p.includeQueryPlanInFetchConfiguration {
+		return
+	}
+	query, err := astprinter.PrintStringIndent(operation, p.upstreamSchema, "  ")
```

Comment:
p.upstreamSchema is not used - we could remove it from planner and pass here just nil

Reply:
removed second arg from printer